### DISCUSSION TITLE
APEC() Normalisation

### DISCUSSION THREAD

**Discussion by Fred-Jennings** [threeML/threeML#600] [Q&A] [answered]
Hi there, I've just started using threeML along with XSPEC models to fit some cluster emission. This is probably a dumb question so apologies if there is something obvious I'm missing, but basically I am trying to figure out how the apec normalisation given after the fit corresponds to the normalisation described under https://heasarc.gsfc.nasa.gov/xanadu/xspec/manual/XSmodelApec.html, such that I can then use it directly to find the electron density? The units associated with the fit normalisation are 1/(kev s cm^2) which doesn't match up with the units of (1/DA) * EM, so how should one convert from one to the other?

**Comment by grburgess** [comment, marked as answer]
APEC is a contributed model, so I'm not an expert on why the normalization is like this, however, according to its display and doctoring, the normalization should have the same value as the XSPEC model. 

I think the issue is that in the units specification, the code specifies the units as having  1/(kev s cm^2). This is probably a work around to allow this type of normalization. I think this can be written better so that the normalization reflects its actual unit, it just requires some rewrite of the code. I will create an issue in astromodels to keep track of this.

**Reply by Fred-Jennings** [reply]
Okay, that makes sense. Thank you very much for clearing that up. Just a final question I had if that's okay, raised whilst I was thinking about these units; do you know if the normalisation (or any other fit value for that matter) will explicitly depend on the area of the image on the detector, and if so where threeML gets this info from in the .pha? I have observation cut-outs from the whole image where no change has been made as of yet to any header(s) describing "area", and I want to make sure that this isn't affecting the fits in any way.

**Reply by grburgess** [reply]
It should not. The area is only passed in the x ray plug-ins through the scale factors read in from the PHA files.

**Reply by Fred-Jennings** [reply]
Great, thanks. Are these scale factors like ["AREASCAL"] etc from the pha? Do you know what they actually refer to, and if threeml will use them for anything? (Apologies, I have tried to find this online, but cannot find anywhere a good description of these headers)

**Reply by grburgess** [reply]
These things are defined by the OGIP standard: https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/spectra/ogip_92_007/node6.html

and they will be in the headers of PHA files. They define things like the fraction of area of the source and background region and are used to scale the counts along with he timing exposures. These are fixed quantities that are precalculated when loading the data and then multiplied on the model side of the calculation when folding a model through the response so that the model gets scaled to the observation. this will be the cm^-2  part. So data are just in counts / keV. The model comes in the for photons / keV / cm2 /s. The model is folded thru the matrix and photons -> counts and then hit with the exposure  and scale. This converts model to the data units counts / keV (or per energy bin) so that the likelihood is in the proper form of counts in each bin.

**Reply by Fred-Jennings** [reply]
Amazing, that clears it up. Thanks so much for all the help!

**Reply by grburgess** [reply]
No worries.

**Comment by grburgess** [comment]
raised here: https://github.com/threeML/astromodels/issues/198
